CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+Contributing to Ansible-Satellite6
+==================================
+
+To contribute to ansible-satellite6, please use pull requests on a branch of your own fork.
+
+After creating your fork on github, you can do:
+```
+git clone git@github.com:yourname/ansible-satellite6
+cd ansible-satellite6
+git checkout -b your-branch-name
+DO SOME CODING HERE
+git add your new files
+git commit
+git push origin your-branch-name
+```
+You will then be able to create a pull request from your commit. 
+
+All fixes or new addition to the project should be accompanied by tests that 
+should PASS. 
+
+Feel free to raise issues in the repo if you feel a fix or new feature support is needed.
+
+Standards
+=========
+Install and Run [ansible-lint](https://github.com/willthames/ansible-lint) against your code changes.
+
+Also, Install and Run [ansible-review](https://github.com/willthames/ansible-review) against your code changes.
```

Comment:
We may want to consider conforming to ansible-galaxy init --offline when creating roles as well.

Reply:
Yup, will be sending out a mail about all the best-practices stuff soon.